docs-src/http-routing.md

```diff
@@ -0,0 +1,63 @@
+# HTTP routing
+
+HTTPRoute resources allow you to match on HTTP traffic and direct it Kubernetes
+backends. This guide shows how the HTTPRoute matches on host, header, and path
+fields and forwards traffic to different Kubernetes Services.
+
+The following diagram describes a required traffic flow across three different
+Services:
+
+- Traffic to `foo.example.com/login` is forwarded to `foo-svc` 
+- Traffic to `bar.example.com/*` with a `Env: canary` header is forwarded 
```

Comment:
Nit for both this and the corresponding graphic + yaml, I think I'd prefer the lowercase `env` here.